graylog2-web-interface/src/views/components/contexts/SearchPagePreferencesState.tsx

```diff
@@ -28,9 +28,9 @@ import { PreferencesActions } from 'stores/users/PreferencesStore';
 import type User from 'logic/users/User';
 
 type Props = {
-  children: (layoutConsumer: {
-    setLayoutState: (stateKey: string, value: boolean) => void,
-    getLayoutState: (stateKey: string, defaultValue: boolean) => boolean,
+  children: (preferencesConsumer: {
+    setPreferences: (stateKey: string, value: boolean) => void,
```

Comment:
I think we should name this `setPreference`, because we are sitting a single preference and not all of them. Same for `getPreference(s)`.